web_responsive/models/res_users.py

```diff
@@ -24,3 +24,11 @@ class ResUsers(models.Model):
         default="milk",
         required=True,
     )
+    is_redirect_home = fields.Boolean(
+        string="Redirect to Home",
+        help="Automatically navigate to " "the home dashboard after signing in",
+    )
+
+    @api.onchange("action_id")
+    def _onchange_action_id(self):
+        self.is_redirect_home = self.action_id and self.is_redirect_home
```

Comment:
I don't understand this onchange.

Reply:
@pedrobaeza 
Apply the change to the code as described above. This onchange method will set the value of is_redirect_home to False when the user sets the Home Action under Preferences.

Reply:
OK, although this can be implemented as a computed writable field for avoiding that this works only by UI. The commit message shouldn't contain the module version, and there's still a comment unattended.

Reply:
@pedrobaeza 
Updated as per the suggestion.

Reply:
@pedrobaeza Why do you prefer a compute field instead of a constraint?

Reply:
Both are complementary. I only prefer computed writable over onchange.